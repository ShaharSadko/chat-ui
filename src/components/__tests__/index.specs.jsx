import App from '../index'
import React from 'react';
import {shallow} from 'enzyme';
const socket = {};

describe('src/components/index', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<App socket={socket}/>);
        expect(wrapper).toMatchSnapshot();
    });

});