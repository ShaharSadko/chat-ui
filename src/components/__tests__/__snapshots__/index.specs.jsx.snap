// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/index should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App
    socket={Object {}}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Container
          className="spotim-header"
>
          <div
                    className="spotim-title"
          >
                    Welcome to the Spot.IM Chat app
          </div>
          <div>
                    <styled.div>
                              <Image
                                        as="img"
                                        size="tiny"
                                        src="spotim-logo.jpg"
                                        ui={true}
                              />
                    </styled.div>
          </div>
</Container>,
        <Connect(MessageListArea)
          socket={Object {}}
/>,
        <Connect(MessageCreationArea)
          socket={Object {}}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <div
              className="spotim-title"
>
              Welcome to the Spot.IM Chat app
</div>,
            <div>
              <styled.div>
                            <Image
                                          as="img"
                                          size="tiny"
                                          src="spotim-logo.jpg"
                                          ui={true}
                            />
              </styled.div>
</div>,
          ],
          "className": "spotim-header",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome to the Spot.IM Chat app",
              "className": "spotim-title",
            },
            "ref": null,
            "rendered": "Welcome to the Spot.IM Chat app",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <styled.div>
                <Image
                                as="img"
                                size="tiny"
                                src="spotim-logo.jpg"
                                ui={true}
                />
</styled.div>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <Image
                  as="img"
                  size="tiny"
                  src="spotim-logo.jpg"
                  ui={true}
/>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "as": "img",
                  "size": "tiny",
                  "src": "spotim-logo.jpg",
                  "ui": true,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "socket": Object {},
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "socket": Object {},
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Container
            className="spotim-header"
>
            <div
                        className="spotim-title"
            >
                        Welcome to the Spot.IM Chat app
            </div>
            <div>
                        <styled.div>
                                    <Image
                                                as="img"
                                                size="tiny"
                                                src="spotim-logo.jpg"
                                                ui={true}
                                    />
                        </styled.div>
            </div>
</Container>,
          <Connect(MessageListArea)
            socket={Object {}}
/>,
          <Connect(MessageCreationArea)
            socket={Object {}}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <div
                className="spotim-title"
>
                Welcome to the Spot.IM Chat app
</div>,
              <div>
                <styled.div>
                                <Image
                                                as="img"
                                                size="tiny"
                                                src="spotim-logo.jpg"
                                                ui={true}
                                />
                </styled.div>
</div>,
            ],
            "className": "spotim-header",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Welcome to the Spot.IM Chat app",
                "className": "spotim-title",
              },
              "ref": null,
              "rendered": "Welcome to the Spot.IM Chat app",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <styled.div>
                  <Image
                                    as="img"
                                    size="tiny"
                                    src="spotim-logo.jpg"
                                    ui={true}
                  />
</styled.div>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Image
                    as="img"
                    size="tiny"
                    src="spotim-logo.jpg"
                    ui={true}
/>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "as": "img",
                    "size": "tiny",
                    "src": "spotim-logo.jpg",
                    "ui": true,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "socket": Object {},
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "socket": Object {},
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
